runtime/lua/vim/lsp/util.lua

```diff
@@ -1156,7 +1154,7 @@ function M.stylize_markdown(bufnr, contents, opts)
           i = i + 1
           break
         end
-        table.insert(stripped, line)
+        stripped[#stripped + 1] = line
```

Comment:
For my own learning: Why change this? Is it just personal preference or is `table.insert` slower?

Reply:
I see. TIL.